### ISSUE TITLE
create script dialog says valid path to visualscript with .gd extension and vice versa

### ISSUE THREAD

**Issue by lordadamson** [godotengine/godot#6878]
**Operating system or device - Godot version:**
Fedora 24 Godot master branch at commit 78d97b060a6873a454e710380cb9ef1bde5e4c65

**Issue description** (what happened, and what was expected):
when creating a script you choose gdscript for example and type a name with the extension .gd and tells you valid path at the bottom of the dialog. but then if you change that to visualscript it would still say valid path.

**Steps to reproduce:**
create a visualscript and give it name.
![screenshot from 2016-10-20 15-38-49](https://cloud.githubusercontent.com/assets/6195060/19562137/e265ce7c-96e3-11e6-9e23-0827f2848a41.png)
change it to gdscript.
![screenshot from 2016-10-20 15-38-53](https://cloud.githubusercontent.com/assets/6195060/19562140/e28cd94a-96e3-11e6-832d-f174ae1bf311.png)
when you click on create.

It happens the other way around too.
![screenshot from 2016-10-20 15-39-00](https://cloud.githubusercontent.com/assets/6195060/19562138/e2739d72-96e3-11e6-9fd9-2797c9e589d5.png)

**Link to minimal example project** (optional but very welcome):

**Comment by volzhs** [comment]
is not this resolved now?

**Comment by bojidar-bg** [comment]
Yeah, GitHub doesn't close issues if you list them like you did in the PR:
```
Fix #7016, #6878 and it supersedes #6883
-> will close on #7016, but not #6878
Fix #7016, fix #6878 and it supersedes #6883
-> all good
```

**Comment by lordadamson** [comment]
@bojidar-bg that's why waffle.io is awesome :stuck_out_tongue:
